main.py

```diff
@@ -13,6 +15,7 @@
     parser = argparse.ArgumentParser()
     parser.add_argument('--binck-transactions', dest='transactions_binck', help='The path to an excel file detailing BinckBank transactions.')
     parser.add_argument('--trading212-transactions', dest='transactions_trading212', help='The path to an csv file detailing Trading 212 transactions.')
+    parser.add_argument('--trading212-dividends', dest='dividends_trading212',help='The path to an csv file detailing Trading 212 dividends.')
```

Comment:
With Trading212 we can export a CSV file that includes transactions and dividends at the same time, this will save us work so we do not need to export 2 files. We don't need this argument.